jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamRunConfiguration.kt

```diff
@@ -135,7 +136,7 @@ class SamRunConfiguration(project: Project, factory: ConfigurationFactory) :
 
     private fun handlerDisplayName(): String? {
         val handler = settings.handler ?: return null
-        return settings.runtime?.let { Runtime.fromValue(it).runtimeGroup }?.let { LambdaHandlerResolver.getInstance(it) }?.handlerDisplayName(handler) ?: handler
+        return settings.runtime?.let { Runtime.fromValue(it).nullable?.runtimeGroup }?.let { LambdaHandlerResolver.getInstance(it) }?.handlerDisplayName(handler) ?: handler
```

Comment:
I dont think we actually need this, since UNKNOWN_TO_SDK will lead to null runtime group anyway